webui/react/src/shared/components/ActionDropdown/ActionDropdown.tsx

```diff
@@ -127,45 +136,24 @@ const ActionDropdown = <T extends string>({
         await onOk();
       }
     } catch (e) {
-      menuClickErrorHandler(e, params.key, kind, id);
+      menuClickErrorHandler(e, key, kind, id);
     }
   };
 
-  const menuItems: MenuProps['items'] = actionOrder
-    .filter((act) => !!onTrigger[act])
-    .map((action) => ({
-      danger: danger?.[action],
-      disabled: disabled?.[action],
-      key: action,
-      label: action,
-    }));
-
-  if (menuItems.length === 0) {
-    return (
-      <div className={css.base} title="No actions available" onClick={stopPropagation}>
-        <Button disabled icon={<Icon name="overflow-vertical" />} type="text" />
-      </div>
-    );
-  }
-
   return children ? (
     <>
       <Dropdown
-        menu={{ items: menuItems, onClick: handleMenuClick }}
+        isContextMenu={isContextMenu != null ? isContextMenu : true}
```

Comment:
nit: this should be equivalent to `isContextMenu ?? true`

Reply:
yeah no idea why I did that lol